Add a view for HTTP headers

edsu opened the pull request:
This commit adds a migration that creates a view of the HTTP headers in the response table. Once the view is in place you can run a query like this without requiring JSON parsing:

```sql
SELECT warc_record_id, name, value FROM http_headers;
```

It can be helpful for identifying for things like:

```sql
SELECT
  value,
  COUNT(*) AS count
FROM http_header
WHERE name = 'content-type'
GROUP BY value
ORDER BY count DESC;
```

```
value                              count
---------------------------------  -----
application/javascript             57
image/png                          11
text/css                           7
text/html; charset=utf-8           6
image/jpeg                         4
image/gif                          4
text/fragment+html; charset=utf-8  3
image/svg+xml                      3
text/plain                         2
text/html; charset=UTF-8           1
```

Closes #24

Florents-Tselai commented:
Two comments

* I think we should prefix views with something like `v_*` and make have in the plural (i.e. `v_http_headers`), and actually, this is only response headers, right? 
* We should start documenting these in a table-like format to keep track of things. I like how Postgres does this. e.g. [pg_stat_activity](https://www.postgresql.org/docs/16/monitoring-stats.html#MONITORING-PG-STAT-ACTIVITY-VIEW)

edsu commented:
Ok, I can adjust the view name and the docs. Our other table names are singular, not plural. I think we should be consistent, and don't have a strong preference either way. Do you?

edsu commented:
That was a good idea to treat requests the same, since they have http headers as well.I've updated this commit to create a view for request records as well, renamed both views to use the `v_` prefix, and (hopefully) improved the documentation along the lines of what you pointed to in the Postgres docs.

Let me know if you have a preference for singular or plural table/view names.

Florents-Tselai commented:
Looks beautiful! 

No strong preference for singular / plural; let's keep it singular. No problem.

edsu commented:
CI let me know I need to reformat :-)
